docs/source/main_classes/trainer.rst

```diff
@@ -119,6 +119,60 @@ TFTrainingArguments
     :members:
 
 
+Logging
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+By default :class:`~transformers.Trainer` will use ``logging.INFO`` for the main node and ``logging.WARNING`` for the
+replica nodes if any.
```

Comment:
```suggestion
By default :class:`~transformers.Trainer` will use ``logging.INFO`` for the main process and ``logging.WARNING`` for the
replicas if any.
```
node = machine (at least in the PyTorch launcher) so we should talk of processes.

Reply:
that's definitely more clear! I got mixed up with with `log_on_each_node`. thank you!